<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPlugin::Internal::UserWizardContactWidget</class>
 <widget class="QWidget" name="UserPlugin::Internal::UserWizardContactWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Address</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="ZipCodes::ZipCodesWidget" name="zipCodesWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Telephone and mail</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblPhone">
         <property name="text">
          <string>Work phone</string>
         </property>
         <property name="buddy">
          <cstring>tel1</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="tel1"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="tel2"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="tel3"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblFax">
         <property name="text">
          <string>Fax</string>
         </property>
         <property name="buddy">
          <cstring>fax</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="fax"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblEmail">
         <property name="text">
          <string>Email</string>
         </property>
         <property name="buddy">
          <cstring>mail</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="mail">
         <property name="placeholderText">
          <string>user@example.com</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Home phone</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mobile phone</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZipCodes::ZipCodesWidget</class>
   <extends>QWidget</extends>
   <header location="global">zipcodesplugin/zipcodeswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
